numpy/linalg/tests/test_linalg.py

```diff
@@ -1620,6 +1620,7 @@ def test_matrix_rank(self):
         assert_equal(matrix_rank(ms), np.array([3, 4, 0]))
         # works on scalar
         assert_equal(matrix_rank(1), 1)
+        assert_equal(matrix_rank(I, tol=0.0), matrix_rank(I, rtol=0.0))
```

Comment:
I'm fairly sure this would not have failed without the fix - would suggest using the explicit example from the issue.

Reply:
Sure! I added a regression test with an example from the issue.